torchok/models/backbones/resnet.py

```diff
@@ -4,245 +4,1319 @@
 Copyright 2019 Ross Wightman
 Licensed under The Apache 2.0 License [see LICENSE for details]
 """
-from typing import Union, List, Dict
+from functools import partial
 
 import torch
 import torch.nn as nn
+from timm.models.helpers import build_model_with_cfg, checkpoint_seq
+from timm.models.layers import BlurPool2d, get_attn, GroupNorm
+from timm.models.resnet import create_aa, make_blocks, BasicBlock, Bottleneck
+from timm.data import IMAGENET_DEFAULT_MEAN, IMAGENET_DEFAULT_STD
 
 from torchok.constructor import BACKBONES
-from torchok.models.modules.blocks.se import SEModule
-from torchok.models.modules.blocks.basicblock import BasicBlock
-from torchok.models.modules.blocks.bottleneck import Bottleneck
-from torchok.models.modules.bricks.convbnact import ConvBnAct
-from torchok.models.backbones import BaseBackbone
-from torchok.models.backbones.utils.helpers import build_model_with_cfg
-from torchok.models.backbones.utils.constants import IMAGENET_DEFAULT_STD, IMAGENET_DEFAULT_MEAN
+from torchok.models.backbones.base_backbone import BaseBackbone
+
+
+__all__ = ['ResNet']
 
 
 def _cfg(url='', **kwargs):
     return {
-        'url': url,
-        'input_size': (3, 224, 224),
-        'pool_size': (7, 7),
-        'crop_pct': 0.875,
-        'interpolation': 'bilinear',
-        'mean': IMAGENET_DEFAULT_MEAN,
-        'std': IMAGENET_DEFAULT_STD,
-        **kwargs
+        'url': url, 'input_size': (3, 224, 224), 'pool_size': (7, 7),
+        'crop_pct': 0.875, 'interpolation': 'bilinear',
+        'mean': IMAGENET_DEFAULT_MEAN, 'std': IMAGENET_DEFAULT_STD,
+        'first_conv': 'conv1', **kwargs
     }
 
 
 default_cfgs = {
-    # ResNet
-    'resnet18': _cfg(url='https://torchok-hub.s3.eu-west-1.amazonaws.com/resnet18-torchok.pth'),
+    # ResNet and Wide ResNet
+    'resnet10t': _cfg(
+        url='https://github.com/rwightman/pytorch-image-models/releases/download/v0.1-rsb-weights/resnet10t_176_c3-f3215ab1.pth',
```

Comment:
Why the links not like HRNet style? url_base_path = 'h[ttps://github.com/rwightman/pytorch-image-models/releases/download/v0.1-hrnet/](https://github.com/rwightman/pytorch-image-models/releases/download/v0.1-rsb-weights)'